opensrp-anc/src/test/java/org/smartregister/anc/library/util/JsonFormUtilsTest.java

```diff
@@ -225,6 +227,106 @@ public void testSaveImageInvokesSaveStaticImageToDiskWithCorrectParams() throws
+    public void testSaveImageInvokesSaveStaticImageToDiskWithCorrectParamsWhereProviderIdAndEntityIdIsNull() throws Exception {
+        PowerMockito.mockStatic(AncLibrary.class);
+        PowerMockito.when(AncLibrary.getInstance()).thenReturn(ancLibrary);
+        PowerMockito.when(ancLibrary.getCompressor()).thenReturn(compressor);
+        PowerMockito.when(compressor.compressToBitmap(ArgumentMatchers.any(File.class))).thenReturn(null);
+
+        PowerMockito.when(ancLibrary.getContext()).thenReturn(context);
+        PowerMockito.when(context.imageRepository()).thenReturn(imageRepository);
+
+
+        PowerMockito.mockStatic(DrishtiApplication.class);
+        PowerMockito.when(DrishtiApplication.getAppDir()).thenReturn("/images");
+
+
+        PowerMockito.mockStatic(FileUtil.class);
+        PowerMockito.when(FileUtil.createFileFromPath(ArgumentMatchers.anyString())).thenReturn(file);
+        PowerMockito.when(file.exists()).thenReturn(true);
+        PowerMockito.when(FileUtil.createFileOutputStream(file)).thenReturn(outputStream);
+
+        org.smartregister.anc.library.util.JsonFormUtils.saveImage(null, null, "filepath/images/folder/location.jpg");
+        Mockito.verify(imageRepository, Mockito.times(0)).add(ArgumentMatchers.any(ProfileImage.class));
+    }
+
+    @Test
+    @PrepareForTest({AncLibrary.class, FileUtil.class, DrishtiApplication.class})
+    public void testSaveImageInvokesSaveStaticImageToDiskWithCorrectParamsWhereFileNotFoundIsThrown() throws Exception {
+        String PROVIDER_ID = "dummy-provider-id";
+        PowerMockito.mockStatic(AncLibrary.class);
+        PowerMockito.when(AncLibrary.getInstance()).thenReturn(ancLibrary);
+        PowerMockito.when(ancLibrary.getCompressor()).thenReturn(compressor);
+        PowerMockito.when(compressor.compressToBitmap(ArgumentMatchers.any(File.class))).thenReturn(bitmap);
+
+        PowerMockito.when(ancLibrary.getContext()).thenReturn(context);
+        PowerMockito.when(context.imageRepository()).thenReturn(imageRepository);
+
+
+        PowerMockito.mockStatic(DrishtiApplication.class);
+        PowerMockito.when(DrishtiApplication.getAppDir()).thenReturn("/images");
+
+
+        PowerMockito.mockStatic(FileUtil.class);
+        PowerMockito.when(FileUtil.createFileFromPath(ArgumentMatchers.anyString())).thenReturn(file);
+        PowerMockito.when(file.exists()).thenReturn(true);
+        PowerMockito.when(FileUtil.createFileOutputStream(file)).thenThrow(FileNotFoundException.class);
+
+        org.smartregister.anc.library.util.JsonFormUtils.saveImage(PROVIDER_ID, DUMMY_BASE_ENTITY_ID, "filepath/images/folder/location.jpg");
+        Mockito.verify(imageRepository, Mockito.times(0)).add(ArgumentMatchers.any(ProfileImage.class));
+    }
+
+    @Test
+    @PrepareForTest({AncLibrary.class, FileUtil.class, DrishtiApplication.class})
+    public void testSaveImageInvokesSaveStaticImageToDiskWithCorrectParamsWhereCompressToBitmapException() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4948315)